cli/cobra.go

```diff
@@ -159,15 +161,27 @@ func isFirstLevelCommand(cmd *cobra.Command) bool {
 	return cmd.Parent() == cmd.Root()
 }
 
-func commandVendor(cmd *cobra.Command) string {
-	width := 13
-	if v, ok := cmd.Annotations[pluginmanager.CommandAnnotationPluginVendor]; ok {
-		if len(v) > width-2 {
-			v = v[:width-3] + "…"
+func isPluginFirstLevelCommand(cmd *cobra.Command) bool {
+	return isPlugin(cmd) && isFirstLevelCommand(cmd)
+}
+
+func decoratedCommandName(cmd *cobra.Command) string {
+	decoration := " "
+	if isPluginFirstLevelCommand(cmd) {
```

Comment:
Was just looking at this again, but I think we're only interested if a command is provided by a plugin (irregardless if it's a subcommand?). If this code is called from a plugin itself, it would not see itself as a plugin, whereas if it's called from the main CLI, if would just add the decoration. If correct, this could be changed to:

```suggestion
	if isPlugin(cmd) {
```

Reply:
Done